src/lib/components/payload-input.svelte

```diff
@@ -1,31 +1,20 @@
-<script context="module" lang="ts">
-  const encoding = ['json/plain', 'json/protobuf'] as const;
-  export type PayloadInputEncoding = (typeof encoding)[number];
-  export const isPayloadInputEncodingType = (
-    x: any, // eslint-disable-line @typescript-eslint/no-explicit-any
-  ): x is PayloadInputEncoding => encoding.includes(x);
-</script>
-
 <script lang="ts">
-  import { type Writable } from 'svelte/store';
-
   import { onDestroy } from 'svelte';
+  import { v4 } from 'uuid';
```

Comment:
I sometimes like to do `import { v4 as id } from 'uuid'` or `import { v4 as uuid } from 'uuid'` because I always forget what `v4` is supposed to be. 😆